opengovsg/plumber
# PLU-84: Add fix for delay + allow time configuration

TODO:
- [x] Add test cases for the different possible combinations for date time

## Problem

Delay with variables does not work.

## Solution

Use computed parameters instead.
Also, allow users to configure time as well for the delay as an optional field.
Stick to using date instead of luxon datetime because the JS date accepts more types of format which sounds better to me

## Before & After Screenshots

**BEFORE**:
<img width="874" alt="image" src="https://github.com/opengovsg/plumber/assets/65110268/8b71b61e-63a4-4beb-89d7-0df14ca2fe62">

**AFTER**:
<img width="906" alt="image" src="https://github.com/opengovsg/plumber/assets/65110268/cef74f9a-4d70-4303-948a-58344ef20325">


## Tests
- Check whether pipe can be published.
- Check whether the delay works without setting time.
- Check whether the delay works with the time set.
- Check whether error works for delay for/until if invalid input is entered
